cpp/src/transform/transform.cpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2019, NVIDIA CORPORATION.
+m* Copyright (c) 2019, NVIDIA CORPORATION.
```

Comment:
```suggestion
 * Copyright (c) 2019, NVIDIA CORPORATION.
```

Reply:
Yes, and that's why the style check is passing, but will fix this one